### PR TITLE
Nest author in first loop block, rather than outside

### PR DESCRIPTION
**Pull request by akmiller01**
This will require some testing on dev first, but in theory it should end up looking like the pattern library.

### CODE REVIEW

File: di_website/templates/publications/publication_chapter_page.html [hunk 1]
```diff
@@ -22,21 +22,6 @@
       {% include 'publications/chapter-nav/nav.html' with page=page.get_parent.specific menu_label='Contents' chapter_number=0 %}
     {% endif %}
 
-    {% if page.authors|length > 0 %}
-      <section class="section section--large">
-        <div class="chapter-block-wrapper">
-          <div class="row row--narrow">
-            <div class="larger-center">
-              <div class="chapter-block__item">
-                <div class="chapter-block_content">
```
Comment (edwinmp) [inline di_website/templates/publications/publication_chapter_page.html]: Strange, I thought I'd fixed this a long while back ... As far as I can tell, this class `chapter-block_content` is responsible for the difference with the PL. They should have updated this as well, or removed the styles completely. In fact, I think this is where the change should originate, not on our end ... the code below should be updated to match all else.
`
.chapter-block_content .m-authors {
    background-color: transparent;
    padding: 0;
    padding-left: 2em;
    border-left: 2px solid #ddd;
}
`

**Comment by edwinmp** [comment]
@akmiller01 is there anything pending on this?

**Comment by akmiller01** [comment]
I was waiting on approval of the appearance from @AnnaHope there are screenshots in issue 663.

**Comment by AnnaHope** [comment]
@akmiller01 really sorry, didn't mean to hold this up! The screenshots look good to me so happy for you to make this live.

**Pull request merged**